packages/client/src/contexts/Web3.js

```diff
@@ -4,6 +4,13 @@ import networks from "../networks";
 import { useRouteMatch } from "react-router-dom";
 import { useFclUser, useTreasury, useNFTs } from "../hooks";
 
+export const REGULAR_LIMIT = 55;
+export const CREATE_TREASURY_LIMIT = 80;
+export const UPDATE_SIGNER_LIMIT = 110;
+export const SIGNED_LIMIT = 300;
```

Comment:
just curious for those actions with SIGNED_LIMIT, are they sharing common complexity of contract code? If there is any chance that certain action need to adjust limit but not for others, we might want to separate them now so to have the flexibility for limit update in future.

Reply:
Proposing all actions has the same checks in the contract code (is the signature valid, is it the signer on the treasury etc), and similar computation effort is used, so the similar limit should be there